eth_worker/eth_manager/celery_tasks.py

```diff
@@ -80,7 +80,8 @@ def synchronize_third_party_transactions(self):
 
 @celery_app.task(**low_priority_config)
 def add_transaction_filter(self, contract_address, contract_type, filter_parameters, filter_type, decimals = 18, block_epoch = None):
-    return blockchain_sync.add_transaction_filter(contract_address, contract_type, filter_parameters, filter_type, decimals, block_epoch)
+    blockchain_sync.add_transaction_filter(contract_address, contract_type, filter_parameters, filter_type, decimals, block_epoch)
+    return True
```

Comment:
@michielderoos This is the filter I was referring to (returning boolean instead of the filter). Is this correct?

Reply:
There are 3 other definitions of `add_transaction_filter` in other files that return the filter.

Reply:
The reason for this is that `blockchain_sync.add_transaction_filter` returns a `TransactionFilter` object, which isn't JSON serializable, and Celery expects things directly returned from its objects to be serializable.  
I did make a change here though-- now I'm building a serializable object containing the filter object's data. Potentially useful if we ever want to change filter creation to a sync call.

Reply:
Okay thanks, it just stood out as the only function of that name returning boolean.
@michielderoos Will you be adding automated tests to show a blockchain tx updating a balance correctly in the db?
What is left before this PR can be merged?